### DISCUSSION TITLE
There a way I can use mini.notify to display diagnostics

### DISCUSSION THREAD

**Discussion by olekspickle** [mrcjkb/rustaceanvim#769] [Q&A]
### Feature description

I want to setup rustaceanvim and display all the diagnostics to use mini.notify

**Comment by mrcjkb** [comment]
Hey :wave: 

This plugin uses Neovim's built-in LSP client to provide diagnostics.
For integrating with mini.notify, you're probably better off asking there.

**Comment by olekspickle** [comment]
If anyone else would like to make it work - mini.notify just works without any config, albeit not as pretty but you know...minimal!
For prettier looking stuff - [noice](https://github.com/folke/noice.nvim) with nui does exactly what you'd want, whether you are a mini cult member or an OG `notify.nvim` user
